modules/cli/src/main/scala/scala/cli/commands/InstallHome.scala

```diff
@@ -0,0 +1,39 @@
+package scala.cli.commands
+
+import caseapp._
+import coursier.env.{EnvironmentUpdate, ProfileUpdater}
+
+import java.io.File
+import scala.util.Properties
+
+object InstallHome extends ScalaCommand[InstallHomeOptions] {
+  override def hidden = true
+  override def names = List(
+    List("install", "home")
+  )
+  def run(options: InstallHomeOptions, args: RemainingArgs): Unit = {
+
+    if (Properties.isWin)
+      System.err.println(s"Install home is not supported yet on windows")
```

Comment:
NIT
```suggestion
      System.err.println("Install home is not supported yet on Windows")
```

Reply:
But we could make that work on Windows I think, just like in [`AddPath`](https://github.com/VirtuslabRnD/scala-cli/blob/d324ee94766bbbf585cab6304e4044dd9d117cd5/modules/cli/src/main/scala/scala/cli/commands/AddPath.scala#L21-L28).

Reply:
Good point, we should also support windows